infra/gcp/ensure-release-projects.sh

```diff
@@ -36,6 +36,7 @@ function usage() {
 PROJECTS=(
     k8s-release
     k8s-release-test-prod
+    releng-test
```

Comment:
```suggestion
```

Instead, add this as `releng-test` in the staging script.

Example:
https://github.com/kubernetes/k8s.io/blob/dd52648354d2c485268f5717de7d49eefc085403/infra/gcp/ensure-staging-storage.sh#L97